File: intro_scala/book-exercises/5-BottleDeposits.py
##
# Compute the refund amount for a collection of bottles.
#

LESS_DEPOSIT = 0.10
MORE_DEPOSIT = 0.25

less = int(input("how many containers 1 litre or less do you have?"))
more = int(input("How many containers more than 1 litre do you have?"))

#Compute the refundnamount
refund = less * LESS_DEPOSIT + more + MORE_DEPOSIT

#Display the result
print("Your total refund  wil be $%.2f." % refund)

File: intro_scala/book-exercises/5-BottleDeposits.scala
/*
Compute the refund amount for a collection of bottles
*/
import io.StdIn._

//Read input from user
@main
def inputRead():Unit = {
    val LESS_DEPOSIT = 0.10
    val MORE_DEPOSIT = 0.15
    val less = readInt
    val more = readInt
    val refund = less*LESS_DEPOSIT + more*MORE_DEPOSIT
    println("Your total refund will be " + refund)
}
